docs/jax.experimental.global_device_array.rst

```diff
@@ -0,0 +1,11 @@
+jax.experimental.global_device_array module
+===========================================
+
+.. automodule:: jax.experimental.global_device_array
+
+API
+---
+
```

Comment:
I wonder if we should add a table of contents? The sidebar doesn't include information on what's on this page. Check out https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#table-of-contents

Reply:
I'll do this in a follow up.